testsuite/integration-arquillian/tests/base/testsuites/fips-suite

```diff
@@ -11,10 +11,6 @@ org.keycloak.testsuite.x509.**
 MutualTLSClientTest
 FAPI1Test
 FAPICIBATest
-KcRegTest
```

Comment:
This needs the `securerandom.strongAlgorithms=PKCS11:SunPKCS11-NSS-FIPS` at JDK level because it's using the `kcreg.sh`. In my branch I'm chaging it at JDK level to make it pass.